api/auth/authproviders.py

```diff
@@ -66,7 +66,7 @@ def validate_code(self, code, **kwargs):
             'access_token': code,
             'uid': uid,
             'auth_type': self.auth_type,
-            'expires': datetime.datetime.utcnow() + datetime.timedelta(hours=1)
+            'expires': datetime.datetime.utcnow() + datetime.timedelta(days=14)
```

Comment:
That is the JWT expiry for our server we are using. JWTs inherently can't be "refreshed". New work would need to be done to support a flow that "re-checks" rather than refreshes. That can be addressed in a separate PR. This change is to prevent users from being logged out every hour when their JWT token really lasts 14 days.